mslice/workspace/base.py

```diff
@@ -0,0 +1,48 @@
+import abc
+
+
+class WorkspaceBase(object):
+    __metaclass__ = abc.ABCMeta
+
+    def __init__(self, mantid_workspace):
```

Comment:
I think we would be better leaving `__init__` out completely as this is now an interface so it would be better that it has no implementation.